registry/storage/linkedblobstore.go

```diff
@@ -95,10 +97,56 @@ func (lbs *linkedBlobStore) Put(ctx context.Context, mediaType string, p []byte)
 	return desc, lbs.linkBlob(ctx, desc)
 }
 
+type createOptions struct {
```

Comment:
Sure thing. I'll extend the docs for BlobCreateOptions and then add some minor explanation to both createOptions